### ISSUE TITLE
Provide helm values via secrets or configmaps

### ISSUE THREAD

**Issue by thecodeassassin** [projectsveltos/addon-controller#516]
**Is your feature request related to a problem? Please describe.**

When you have many Helm charts to install that require a lot of values the only way to pass these values is via the "values" parameter. 

**Describe the solution you'd like**
It would be great if a reference can be given to a configmap or secret so that it's easier to both provide secret values to a chart but also help organise the ClusterProfiles.

A similar functionality is "PolicyRef"

  policyRefs:
  - name: requirements
    namespace: default
    kind: ConfigMap
    
 This works really great for one-off yaml files that need to be applied such as CRDs.

**Describe alternatives you've considered**
There isn't really a great alternative at the moment. One can install Helm Controller (https://fluxcd.io/flux/guides/helmreleases/#refer-to-values-in-secret-generated-with-kustomize-and-sops) and manage helm charts separately but this just adds more tools to manage and keep an eye on.

**Additional context**

**Comment by gianlucam76** [comment]
Proposing this:

```
	// Values field allows to define configuration for the Helm release.
	// These values can be static or leverage Go templates for dynamic customization.
	// When expressed as templates, the values are filled in using information from
	// resources within the management cluster before deployment.
	// +optional
	Values string `json:"values,omitempty"`

	// ValuesFrom can reference ConfigMap/Secret instances. Within the ConfigMap or Secret data,
	// it is possible to store configuration for the Helm release.
	// These values can be static or leverage Go templates for dynamic customization.
	// When expressed as templates, the values are filled in using information from
	// resources within the management cluster before deployment.
	// +optional
	ValuesFrom []ValueFrom `json:"valuesFrom,omitempty"`
```

with

```yaml
type ValueFrom struct {
	// Namespace of the referenced resource.
	// For ClusterProfile namespace can be left empty. In such a case, namespace will
	// be implicit set to cluster's namespace.
	// For Profile namespace must be left empty. The Profile namespace will be used.
	Namespace string `json:"namespace"`

	// Name of the referenced resource.
	// +kubebuilder:validation:MinLength=1
	Name string `json:"name"`

	// Kind of the resource. Supported kinds are:
	// - ConfigMap/Secret
	// +kubebuilder:validation:Enum=ConfigMap;Secret
	Kind string `json:"kind"`
}
````

**Comment by gianlucam76** [comment]
Fixed. Will be available in v0.29.0
